### DISCUSSION TITLE
Nfix simulations

### DISCUSSION THREAD

**Discussion by anddebol** [ARM-Development/comble-mip#49] [General]
Hi everyone, quick question. For the experiments with "Fixed droplet Nd= 20 cm^-3", what  is Nd precisely?  Judging by output_les/dharma/sandbox/DHARMA_Lx25_dx200_FixN.nc (or available WRF output) it is not a cloud droplet number concentration as none of n(lc/lcic) is a constant in the output. Is it a fixed fraction of CNN or fixed CNN then?

**Comment by AlejandroBaroPerez** [comment]
Hi @anddebol , I asked the same to @fridlind  a few days ago. She explained me that, for the Nfix simulations, if there is any cloud water present, then droplet number concentration is assumed to be the fixed value (Nd = 20 cm^-3). However, the variable "nlc" that is requested is the average number concentration over all grid cells ("all sky" per the notes field here), so it is often less than the value corresponding to 20/cm3 because of non-cloudy grid cells contributing 0s.

**Comment by fridlind** [comment]
Hi @anddebol and @AlejandroBaroPerez: The nlc and nlcic variables requested are mixing ratios over all grid cells and cloudy grid cells, respectively. When accounting for the vertical variation in the density of air, nlcic does correspond to a fixed number concentration of 20 cm^-3. If this does not resolve the confusion, please do keep the questions coming in case we overlooked anything. Many thanks!

**Comment by anddebol** [comment]
yes, but from what I checked nlcic is also not a constant in those outputs.

**Comment by anddebol** [comment]
![nclic](https://github.com/ARM-Development/comble-mip/assets/4727244/afd25dbf-3e11-4eb8-8f07-823db8b33c52)
This is from dharma_Fixn nclic output. I guess that discrepancy ( judging by the layered structure) might come from different LWC definitions (particularly using rho_ref in one place and density from ideal gas law in another). This is just a guess though.

**Comment by anddebol** [comment]
Ohh, my mistake, I miss the kg-1 part, sorry for the confusion.

**Reply by fridlind** [reply]
No worries. We very much appreciate everyone alerting us to possible inconsistencies sooner rather than later!
